src/Features/CSharp/Portable/Diagnostics/Analyzers/CSharpTypeStyleDiagnosticAnalyzerBase.cs

```diff
@@ -119,7 +118,7 @@ private bool ShouldAnalyzeVariableDeclaration(VariableDeclarationSyntax variable
 
             return isSupportedParentKind &&
                    variableDeclaration.Variables.Count == 1 &&
-                   variableDeclaration.Variables.Single().Initializer.IsKind(SyntaxKind.EqualsValueClause);
```

Comment:
SeparatedSyntaxList already has First/Last.  I would prefer we just add Single to it as well.  If this goes into 15.6 that would be no problem to do.